pkg/mfa/cookie.go

```diff
@@ -0,0 +1,27 @@
+package mfa
+
+import (
+	"net/http"
+
+	"github.com/authgear/authgear-server/pkg/lib/config"
+	"github.com/authgear/authgear-server/pkg/util/httputil"
+)
+
+const CookieName = "mfa_device_token"
+
+type CookieDef struct {
+	Def *httputil.CookieDef
+}
+
+func NewDeviceTokenCookieDef(r *http.Request, cfg *config.AuthenticationConfig) CookieDef {
+	def := &httputil.CookieDef{
+		Name:     CookieName,
+		Path:     "/",
+		SameSite: http.SameSiteLaxMode,
```

Comment:
I think it can be `Strict`, we expect the cookie to be used through navigation within the auth UI only.